yt/utilities/amr_kdtree/amr_kdtree.py

```diff
@@ -343,7 +343,9 @@ def get_brick_data(self, node):
             )
             for i, field in enumerate(self.fields):
                 if self.log_fields[i]:
-                    dds.append(np.log10(vcd[field].astype("float64")))
+                    v = vcd[field].astype("float64")
```

Comment:
This is the one of two code changes. It prevents a warning to be raised by numpy due to trying to compute the log of a negative value.

Reply:
good catch